### PR TITLE
feat(technical-user-details): enable credential reset

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

add reset credential button and respective api integration

## Why

enable new functionality

## Issue

NA

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Comment by manojava-gk** [comment]
@evegufy  can you merge this PR? Lavanya has already approved it

File: src/assets/locales/de/main.json [hunk 1]
```diff
@@ -738,6 +739,8 @@
         "confirmDeleteNotificationTitle": "Das Löschen des Benutzers war erfolgreich.",
         "confirmDeleteNotificationDescription": "Die Aktion kann nicht rückgängig gemacht werden. Bitte erstellen Sie einen neuen Benutzer, falls dies erforderlich ist",
         "deleteTechUserNotificationErrorTitle": "Something went wrong",
+        "resetCredentialError": "Something went wrong",
+        "resetCredentialSuccess": "Reset credential is successful",
```
Comment (evegufy) [inline src/assets/locales/de/main.json:743]: Could you please change the message here to "Credential reset was successful"?

File: src/assets/locales/en/main.json [hunk 2]
```diff
@@ -737,6 +738,8 @@
         "confirmDeleteNotificationTitle": "User deletion was successful.",
         "confirmDeleteNotificationDescription": "The action can not be reversed. Please create a new user in case it's needed",
         "deleteTechUserNotificationErrorTitle": "Something went wrong",
+        "resetCredentialError": "Something went wrong",
+        "resetCredentialSuccess": "Reset credential is successful",
```
Comment (evegufy) [inline src/assets/locales/en/main.json:742]: Could you please change the message here to "Credential reset was successful"?

File: src/assets/locales/de/main.json (hunk 1 above)
Comment (evegufy) [inline src/assets/locales/de/main.json:743]: Could you please change the message here to "Credential reset was successful"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,8 @@
 
 - Active Subscription
   - Switch endpoint used to display "App Subscriptions"
-- Technical User
+- Technical User Details
+  - Reset credential
```
Comment (evegufy) [inline CHANGELOG.md:8]: Could you please change to "Enable credential reset"?
I think that leaves less room for interpretation.

**Review by evegufy — CHANGES_REQUESTED** [review]
Please also adjust the 'Why' in the PR description: 'new request' isn't very helpful description, at this point simply 'enable new functionality' I'd consider a more helpful information.

**Review by evegufy — CHANGES_REQUESTED** [review]
My comments regarding success message weren't implemented

**Review by evegufy — APPROVED** [review]

**Pull request merged**